spec/javascripts/reducers/pastIterations_spec.js

```diff
@@ -0,0 +1,128 @@
+import actionTypes from 'actions/actionTypes';
+import reducer from "../../../app/assets/javascripts/reducers/pastIterations";
+
+describe("Past Iterations Reducer", () => {
```

Comment:
There are mixed use of single and double quotes,
Unless the linter is specifying the contrary, I suggest always using single quotes.